kube/pod.go

```diff
@@ -44,14 +44,20 @@ func NewPodTemplate(role *model.InstanceGroup, settings ExportSettings, grapher
 	spec.Add("dnsPolicy", "ClusterFirst")
 	spec.Add("volumes", getNonClaimVolumes(role, settings.CreateHelmChart))
 	spec.Add("restartPolicy", "Always")
-	if role.Run.ServiceAccount != "" {
+	if role.Run.ServiceAccount != "default" {
 		// This role requires a custom service account
 		block := helm.Block("")
 		if settings.CreateHelmChart {
 			block = helm.Block(authModeRBAC)
 		}
 		spec.Add("serviceAccountName", role.Run.ServiceAccount, block)
 	}
+
+	// Note: This group's PSP reference is not used, as it is not
```

Comment:
While PSP's are on jobs all the PSPs referenced by the jobs of an instance group are merged together into a single policy for the entire group. Because all the jobs currently still run in a single container/pod with a single PSP attached to it.

Will check if I can phrase it better.

Reply:
Ok, checked. This comment is out of date. It harkens back to an interim state where the instance group structure actually had a PSP field. Then this moved to Jobs, and the IG's psp is computed (see `groupPodSecurityPolicy`), and then immediately moved to the associated service account.

Nothing is stored in the IG any longer.

I will remove this comment.

Reply:
Changed in commit 273e9337f26bc193b4da76b7078ba57acae637ef